CHANGELOG.next.toml

```diff
@@ -22,3 +22,11 @@ message = "Fix incorrect redaction of `@sensitive` types in maps and lists."
 references = ["smithy-rs#3765",  "smithy-rs#3757"]
 meta = { "breaking" = false, "tada" = false, "bug" = true, "target" = "client" }
 author = "landonxjames"
+
+[[smithy-rs]]
+message = """
+Add support for <a href="https://smithy.io/2.0/additional-specs/protocols/smithy-rpc-v2.html">the Smithy RPC v2 CBOR protocol</a> both on the client and on the server.
```

Comment:
Do we want to already advertise this? We should at least test event streams work correctly before that. Not saying that they don't already, but I didn't add a test in my PR checking so and this one doesn't either.

Reply:
840f61a added disclaimer in the release notes.